src/ui/menu.c

```diff
@@ -77,12 +64,10 @@ void ui_menu_main() {
         ux_stack_push();
     }
 
-    if (G_coin_config->bip32_pubkey_version == BIP32_PUBKEY_VERSION_MAINNET) {  // mainnet
+    if (BIP32_PUBKEY_VERSION == BIP32_PUBKEY_VERSION_MAINNET) {  // mainnet
```

Comment:
This doesn't need to be done at runtime, maybe you could use something like
`#if coin == bitcoin
#elif coin == bitcoin_test`

Reply:
Since they are constants, the compiler will optimize it away anyway.
But maybe it's indeed more standard to use #ifdefs, I'll do it in a future cleanup.